### PR TITLE
breast cancer model updates

### PR DESCRIPTION
**Pull request by yongqx**

### CODE REVIEW

File: docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 1]
```diff
@@ -394,6 +396,9 @@ Using OR value of 1.89 and P as 0.3
     - I'm wondering if the upper and lower limits of the truncated normal distributions should be narrower? What we are modelling here are the 'guideline times' to next screening, hence shouldn't they fall within the bounds of 1 year or 2 years according to the screening tree? Currently for someone who is in branch A, B, or C and supposed to have yearly screens, their next scheduled screen can be as far in the future as two years (with an upper bound of 700 days). 
 
     - I'm wondering if the Marketscan data, where we got the empirical distributions from, is giving us the time interval between screens that the patient actually showed up to? (which in our model is a combintation of time to next scheduled screening + probabiltiy of showing up)
+    
+    - For individuals survive more than 10 years after the diagnosis and treatment, 
+    send them to susceptible state or we could create a remission state and assume they will never have a chance to be screened during the simulation.
```
Comment (rmudambi) [inline docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst:401]: They are going into a remission state and then continuing to get screened in the model (though the result of the screening will always say in remission).

File: docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 2]
```diff
@@ -310,6 +310,8 @@ Breast cancer screening algorithm was derived from the 2019 guidelines from the
  
 In initialization, We assume that no one has prior knowledge of their DCIS or BC status. Hence no one will be initialized into branch B or C at initialization. Subsequently, if DCIS or LCIS was detected.
 
+For individuals with breast cancer (regardless of detection) and have survived more than 10 years, they are considered to step into a recovered state (R) and should not make a second critical illness (CI) claim. We assume that people in breast cancer state and recovered state follow exactly the same screening algorithm, namely branch A, B, or C depending on their age, sex, and family history status. Note that the negaive screening results were expected for those in R state in order to avoid double counting the CI claim from detected breast cancer.
```
[the remark this answers is not in the document]
Comment (aflaxman) [reply docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst:313]: suggested text: "For individuals in disease state BC (regardless of detection) they have a transition rate of 0.1 (per person-year) of moving into a recovered state (R); this results in an average duration in state BC of 10 years. Individuals do not ever make a second BC CI claim."

File: docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst (hunk 1 above)
Comment (aflaxman) [reply docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst:401]: This keeps things simple, and also lets us count how much screening is happening.

**Comment by yongqx** [comment]
> Approved after updating with Abie's text suggestions.

Yep, i did it!

**Pull request merged**